packages/plugin-amplitude-node/lib/index.ts

```diff
@@ -1,13 +1,24 @@
 /* eslint-disable no-unused-vars, class-methods-use-this */
 import {
   Event,
+  EventOptions,
+  EventMetadata,
   Properties,
   Plugin,
 } from '@itly/sdk';
-import Amplitude, { AmplitudeOptions } from 'amplitude';
+import Amplitude, { AmplitudeOptions, AmplitudeIdentifyResponse, AmplitudeTrackResponse } from 'amplitude';
 
 export { AmplitudeOptions };
 
+export type AmplitudeCallback = (response: AmplitudeIdentifyResponse | AmplitudeTrackResponse) => void;
```

Comment:
I noticed some discrepencies last night between different method calls and unfortunately I think the safest/most fexible thing to do is to create separate interfaces for each method to allow for maimum support instead of least common denominator [see my updated notes in Notion](https://www.notion.so/iteratively/JS-TS-Event-Metadata-f83bdaa6b1814a869c6c41bfb0d97990). Let me know what you think. Definitely open to ideas.